Gems/Atom/RPI/Code/Source/RPI.Public/Pass/FullscreenTrianglePass.cpp

```diff
@@ -35,11 +35,18 @@ namespace AZ
         }
 
         FullscreenTrianglePass::FullscreenTrianglePass(const PassDescriptor& descriptor)
-            : RenderPass(descriptor)
+            : RasterPass(descriptor)
             , m_passDescriptor(descriptor)
         {
             m_defaultShaderAttachmentStage = RHI::ScopeAttachmentStage::FragmentShader;
             LoadShader();
+
+            // Even though this is a RasterPass, it works differently than expected
+            // because it doesn't need draw items from Views. Instead this pass creates its own
+            // draw item (one triangle that covers the whole screen). We set a dummy DrawListTag
+            // with the pass name to avoid warnings from the RPI.
+            auto dummyTag = AZStd::string::format("%s_dummyTag", GetName().GetCStr());
+            SetDrawListTag(AZ::Name(dummyTag));
```

Comment:
I can come up with a better way to use SetDrawListTag(), but FTP, as mentioned above, is technically a Raster Pass. RenderPass is not the proper subclass because ComputePass is also a subclass of RenderPass and ComputePass is not mergeable.

Reply:
Fixed, as explained above.